src/lib/molecules/Event/EventVenue.js

```diff
@@ -1,21 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import TagLink from '../../atoms/TagLink/TagLink';
 
 const EventVenue = (props) => {
   return (
-    // TODO: I don't think this should be an article. It seems like more of an attribute of a larger article, such as the event itself or eventInfo. The things it holds don't seem like paragraphs either.
-    <article className="eventInfo-venue">
+    <div className="eventInfo-venue">
       <h2>Location</h2>
-      <TagLink
+      <a
         href={props.venue.website}
-        title={props.venue.name}
-        elementClass="link-plain"
-      />
+        target="_blank"
+        rel="noopener noreferrer"
+        className="link-plain"
+      >
+        <p>{props.venue.name}</p>
```

Comment:
These are quite a few improvements, and I'm glad you were already on them! However, I'm still a bit concerned about each of these attributes being called paragraphs when most are one to three words. They are more like individual data points and, for SEO's sake, should be grouped in an unordered list (with the bullet point styles removed, of course) or at least put into <span> tags. I actually want to ask Matt about this too though, to make sure we're on the same page.